getting-started/setup-multi-node-basic.md

```diff
@@ -80,6 +30,19 @@ is the host name or IP address of the data node.
 
 The system should be up and running at this point.
 
+## (Optional) Add user roles to the distributed database
+
+After adding your data nodes, it is possible to create and use distributed hypertables,
+but only if running as a superuser common to all nodes. For Timescale Cloud or Forge,
+this means using the created `tsdbadmin` user.
+
+If using a self-managed multi-node cluster, you may wish to create new users with
+multi-node access, or add access to existing roles.  Please refer to the following
+sections depending on the authentication mechanism you are using:
+- Adding roles using [trust authentication][trust_role_setup]
+- Adding roles using [password authentication][password_role_setup]
+- Adding roles using [certificate authentication][certificate_role_setup]
```

Comment:
This section feels odd for Forge and Cloud users, who will stay in the document. I understand that it is for self-managed users who will jump into this file for the add data nodes instructions.

Reply:
Agreed, but it is important for the self-managed users, and hopefully we'll have a way to add users in Forge eventually.  At that point, this section can be expanded to explain how to add setup those roles for multinode.